backend/src/controller/plantings.rs

```diff
@@ -5,17 +5,45 @@
 // FIXME:
 //  Remove these clippy allows when these endpoints are full implemented.
 
-use actix_web::web::Query;
+use std::sync::RwLock;
+
 use actix_web::{
-    delete, get, patch, post,
-    web::{Data, Json, Path},
+    delete, error, get, patch, post,
+    web::{Data, Json, Path, Query},
     HttpResponse, Result,
 };
 
-use crate::db::connection::Pool;
+use crate::config::auth::user_info::UserInfo;
+use crate::model::dto::actions::{
+    CreatePlantActionDto, DeletePlantActionDto, MovePlantActionDto, TransformPlantActionDto,
+};
 use crate::model::dto::{
-    NewPlantingDto, Page, PageParameters, PlantingDto, PlantingSearchParameters, UpdatePlantingDto,
+    NewPlantingDto, PlantLayerObjectDto, PlantingSearchParameters, UpdatePlantingDto,
 };
+use crate::AppDataInner;
+
+/// TODO: REMOVE THIS HACK ❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗❗
+static PLANTINGS: RwLock<Vec<PlantLayerObjectDto>> = RwLock::new(vec![]);
```

Comment:
I think it would be better to leave it as is, for the short while this code remains like this.
When the plant layer DB layer is implemented this gets removed anyway.